pages/platform/databases/postgresql_10_advanced_parameters_references/guide.en-gb.md

```diff
@@ -0,0 +1,115 @@
+| `pg.max_slot_wal_keep_size` | long | PostgreSQL maximum WAL size (MB) reserved for replication slots. Default is -1 (unlimited). wal_keep_size minimum WAL size setting takes precedence over this. |
+| `pg.max_stack_depth` | long | Maximum depth of the stack in bytes |
+| `pg.max_standby_archive_delay` | long | Max standby archive delay in milliseconds |
+| `pg.max_standby_streaming_delay` | long | Max standby streaming delay in milliseconds |
+| `pg.max_wal_senders` | long | PostgreSQL maximum WAL senders |
+| `pg.max_worker_processes` | long | Sets the maximum number of background processes that the system can support |
+| `pg.pg_partman_bgw.interval` | long | Sets the time interval to run pg_partman's scheduled tasks |
+| `pg.pg_partman_bgw.role` | string | Controls which role to use for pg_partman's scheduled background tasks. |
+| `pg.pg_stat_monitor.pgsm_enable_query_plan` | boolean | Enables or disables query plan monitoring |
+| `pg.pg_stat_monitor.pgsm_max_buckets` | long | Sets the maximum number of buckets  |
+| `pg.temp_file_limit` | long | PostgreSQL temporary file limit in KiB, -1 for unlimited |
+| `pg.timezone` | string | PostgreSQL service timezone |
+| `pg.track_activity_query_size` | long | Specifies the number of bytes reserved to track the currently executing command for each active session. |
+| `pg.track_commit_timestamp` | string | Record commit time of transactions. |
+| `pg.track_functions` | string | Enables tracking of function call counts and time used. |
+| `pg.track_io_timing` | string | Enables timing of database I/O calls. This parameter is off by default, because it will repeatedly query the operating system for the current time, which may cause significant overhead on some platforms. |
+| `pg.wal_sender_timeout` | long | Terminate replication connections that are inactive for longer than this amount of time, in milliseconds. Setting this value to zero disables the timeout. |
+| `pg.wal_writer_delay` | long | WAL flush interval in milliseconds. Note that setting this value to lower than the default 200ms may negatively impact performance |
+| `pgbouncer.autodb_idle_timeout` | long | The number of seconds after which - if unused - the automatically created database pools are freed. If set to 0, then timeout is disabled. |
+| `pgbouncer.autodb_max_db_connections` | long | The overall maximum number of server connections per database (regardless of user). Setting it to 0 means it is unlimited. |
+| `pgbouncer.autodb_pool_mode` | string | The PgBouncer pool mode: with session, the server is released back to the pool after the client disconnects; with transaction, the server is released back to the pool after the transaction finishes; with statement the server is released back to the pool after the query finishes (transactions spanning multiple statements are disallowed in this mode). The default value is: session A valid range is: session, transaction, statement |
```

Comment:
Punctuation may need to be improved here, to clarify it all. Maybe using lists or ```<br>```